01-messaging.md

```diff
@@ -225,8 +225,16 @@ Both fields `globalfeatures` and `localfeatures` MUST be padded to bytes with 0s
    * [`gflen*byte`:`globalfeatures`]
    * [`u16`:`lflen`]
    * [`lflen*byte`:`localfeatures`]
+   * [`init_tlvs`:`tlvs`]
```

Comment:
qq on how we define this (i'm not super familiar with the parsing code): intuitively i would think that this would be
```
  * [`tlvs`: `init_tlvs`]
```
where `tlvs` is the type, and `init_tlvs` is the name. the current way is flipped from what I would expect, maybe i'm just not thinking about it correctly?

Reply:
The type is actually defined below... Confusing I know, but it's a consequence of incrementally implementing parsing.

We could just imply TLV from now on, with msg `foo` having a `foo_tlv` implicitly?

Reply:
either works for me, was just curious what the guidelines are :)